lib/icingadb/icingadb-objects.cpp

```diff
@@ -1884,7 +1884,7 @@ void IcingaDB::SendRemovedDowntime(const Downtime::Ptr& downtime)
 		"scheduled_end_time", Convert::ToString(TimestampToMilliseconds(downtime->GetEndTime())),
 		"has_been_cancelled", Convert::ToString((unsigned short)downtime->GetWasCancelled()),
 		"trigger_time", Convert::ToString(TimestampToMilliseconds(downtime->GetTriggerTime())),
-		"cancel_time", Convert::ToString(TimestampToMilliseconds(Utility::GetTime())),
+		"cancel_time", Convert::ToString(TimestampToMilliseconds(downtime->GetCancelTime())),
```

Comment:
And if someone didn’t upgrade both nodes yet and it's 0.. it intendedly doesn’t count as cancelled?

Reply:
To determine if the downtime was cancelled, `has_been_cancelled` should be used. And that one was wrong before already. But yes, for expired downtimes, this will no longer be set to the expire time, but I think this is an acceptable change.

Reply:
I haven't changed anything about this so far. What do you think would be the desired behavior?

Reply:
I'd say the current one is OK.. as long as we know what we do.